recipes/uppasd/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "uppasd" %}
+{% set version = "v6.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UppASD/UppASD/releases/download/{{ version }}/{{ version }}.tar.gz
+  sha256: f483bb6e0a5c75820e4a3d38894c40557548777160eabd3c152c8c356e6ca8ed
+  patches:
+    - binary-name.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}  # [not win]
+    # Windows: complation with flang fails because the provided flang version 5 does not provide
+    # a NORM2 intrinsic; wait for https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/5081
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{ stdlib('c') }}
+    - cmake
+    - ninja
+    - llvm-openmp  # [osx]
+    - libgomp  # [win or linux]
+    - fftw
+  host:
+    - fftw
+    - libblas
+    - liblapack
+
+test:
+  source_files:
+    - examples/SimpleSystems/bccFe
+  commands:
+    - cd examples/SimpleSystems/bccFe && sd  # [not win and build_platform == target_platform]
+    - cd examples/SimpleSystems/bccFe && sd.exe  # [win]
+
+outputs:
+  # For now only distribute the binary but not the static library. Building a shared
+  # library would be preferable but seems to not be straightforward for UppASD (some
+  # things are hardcoded in their CMakeLists.txt).
+  - name: uppasd
+    files:
+      include:
+        - bin/sd  # [not win]
+        - Library/bin/sd.exe  # [win]
```

Comment:
Unfortunately this command name conflicts with an existing package, see https://github.com/conda-forge/sd-feedstock . At the minimum, I guess we need to add a `run_constrained` command to ensure that the two packages are not installed side-by-side, but I would also suggest to open a discussion on Zulip to collect feedback on how to handle this case.

Reply:
That would be great!